### PR TITLE
Users

### PR DESCRIPTION
**Pull request by Moradi66**

### CODE REVIEW

File: frontend/src/components/form/user/addSkill.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            <Label value="Skill level" />
+            <select
+              name="skillLevel"
+              value={this.state.skillLevel}
+              onChange={this.handleChange}
+              className="form-control"
+            >
+              <option>Chose</option>
+              <option value="begganer">Beggener</option>
+              <option value="intermdiat">Intermdiat</option>
```
Comment (KrustyC) [inline frontend/src/components/form/user/addSkill.jsx]: Please change to "Beginner" and "Intermediate"

File: frontend/src/components/header/Navbar.js [hunk 2]
```diff
@@ -58,52 +58,52 @@ export default class NavbarFeatures extends Component {
...
-                  <li className="nav-item user-info-items">
-                    <div className="nav-link">{userName}</div>
-                    <div className="nav-link">
-                      <div className="user-avatar">
-                        {!avatar ? (
-                          <div>
-                            {title === 'Mr' ? (
-                              <img
+              {AuthService.loggedIn ()
+                ? <div>
```
Comment (KrustyC) [inline frontend/src/components/header/Navbar.js]: as you show an empty string if the condition is false, then you could simply do `{ AuthService.loggedIn() && (<div>...</div>)}`

File: frontend/src/components/header/Navbar.js [hunk 3]
```diff
@@ -58,52 +58,52 @@ export default class NavbarFeatures extends Component {
...
+                      &nbsp;
+                      <Button
+                        className="log-out nav-link"
+                        value="Log Out"
+                        onClick={this.onLogOut}
+                      />
+                    </li>
+                  </div>
+                : ''
+                  // <li className="nav-item user-info-items">
```
Comment (KrustyC) [inline frontend/src/components/header/Navbar.js]: please remove if not needed

File: frontend/src/containers/Home.js [hunk 4]
```diff
@@ -1,11 +1,53 @@
...
+      <div className="home-page">
+        <img
+          className="logo-crisis-classroom"
+          alt="Logo crisis classroom"
+          src={CrisisLogo}
+        />
+        <h1 className="welcome">Welcome to Crisis Classroom</h1>
+
+        {!AuthService.loggedIn ()
+          ? <div>
```
Comment (KrustyC) [inline frontend/src/containers/Home.js]: same as above, please use `{!AuthService.loggedIn() && <div>...</div>}`

File: frontend/src/routes/Routes.jsx [hunk 5]
```diff
@@ -40,6 +42,8 @@ class Routes extends Component {
         <PrivateRoute exact path="/template-edited" component={TemplateEdited} />
         <PrivateRoute exact path="/template-deleted" component={TemplateDeleted} />
         <PrivateRoute path="/templates" component={Templates} />
+        <PrivateRoute path="/user-profile" component={UserProfile} />
+        <PrivateRoute path="/get-skill/:id" component={editSkill} />
```
Comment (KrustyC) [inline frontend/src/routes/Routes.jsx]: why is the route called `get-skill` and the page `EditSkill`?

**Comment by KrustyC** [comment]
@Moradi66 there are still a couple of changes that needs to be made

File: frontend/src/components/header/Navbar.js (hunk 3 above)
Comment (Moradi66) [reply frontend/src/components/header/Navbar.js]: I need it later

**Comment by Moradi66** [comment]
What changes???

File: frontend/src/containers/Home.js (hunk 4 above)
Comment (KrustyC) [reply frontend/src/containers/Home.js]: @Moradi66 this change here

**Pull request merged**